mlrun/datastore/targets.py

```diff
@@ -1114,7 +1113,7 @@ def get_spark_options(self, key_column=None, timestamp_key=None, overwrite=True)
         if isinstance(key_column, list) and len(key_column) >= 1:
             spark_options["key"] = key_column[0]
             if len(key_column) > 2:
-                spark_options["sorting-key"] = "_spark_object_sorting_key"
+                spark_options["sorting-key"] = "_spark_object_name"
```

Comment:
Why the change? Isn't this the sorting key rather than the object name?

Reply:
It becomes spark object name